### PR TITLE
Cleanup chains

### PR DESCRIPTION
**Pull request by leanazulyoro**
Clean up all unused chains, leaving only HEMI (not fully enabled because we need contract addresses, otherwise things might break) and HEMI_SEPOLIA

### CODE REVIEW

File: src/addresses.ts [hunk 1]
```diff
@@ -13,228 +11,21 @@ type ChainAddresses = {
...
+  quoterAddress: '0x3899a6090c5C178dB8A1800DA39daD0D06EeEFBE',
+  swapRouter02Address: '0x70523D3F46c9858D3319c4bb02a236Dc6C449d39',
+  tickLensAddress: '0x70AB149e550690D55a46AA326211438c5D47B6D3',
+  v3CoreFactoryAddress: '0xc2f8037a31a4f12b07d20D3f382b1aA03c94519B',
+  v3MigratorAddress: '0x1B1ff6ad47a9fe6eF10b9A235a2B2C8d1CED743e'
+}*/
 
+// Hemi Sepolia addresses
 const HEMI_SEPOLIA_ADDRESSES: ChainAddresses = {
   multicallAddress: '0xf7364E230EA0e5CC41E6Cd4E2a30A26Ca1850b73',
```
Comment (gabmontes) [inline src/addresses.ts:30]: Isn't this one `0xcA11bde05977b3631167028862bE2a173976CA11`? Double-check the rest.

Comment (gndelia) [reply src/addresses.ts:30]: I think this address comes from [here](https://github.com/BVM-priv/uniswap-deploy-v3/blob/57c16ce7535adbf502b1040522a951a8c56060f1/state.json#L3) which was the list that Rohit shared originally when starting to work on Uniswap on Hemi. I don't know which would be now cc @rokso @max-sanchez

Comment (rokso) [reply src/addresses.ts:30]: @gndelia @gabmontes The one I have in uni repo is multicall2 and the one Gabriel has is multicall3.

Comment (gndelia) [reply src/addresses.ts:30]: I don't know which one Uniswaps needs (I'd assume multicall2 ?)

Comment (rokso) [reply src/addresses.ts:30]: For uniswap, you will be using multicall2

**Review by gndelia — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]

**Review by rokso — APPROVED** [review]

**Pull request merged**